FABulous/FABulous_CLI/helper.py

```diff
@@ -131,6 +132,12 @@ def setup_project_env_vars(args: argparse.Namespace) -> None:
         )
         os.environ["FAB_PROJ_LANG"] = args.writer
 
+    # string comparison
+    if os.environ["VERSION"] < version("FABulous-FPGA"):
```

Comment:
I think it's a bit too much simplified, if we just check if project version is not newer than your current fabulous version. Older fabulous versions can't handle newer projects as well.

I'm also not sure, if we should check for the whole version string, technically breaking changes should only happen in major releases. 

 I think we should at least not throw warnings for patch, and dirty versions.

Reply:
I will make the changes once we decide what we want for versioning. Probably using something like [packaging](https://pypi.org/project/packaging/) should allow us to do more fine control.